NSUserReplacementItems.plist

```diff
@@ -2465,7 +2497,7 @@
             <key>replace</key>
             <string>:four:</string>
             <key>with</key>
-            <string>4</string>
```

Comment:
GH doesn't preview this correctly, but it imports correctly:

![Uploading Keyboard 2015-08-05 16-01-44.png…]()